src/olympia/addons/views.py

```diff
@@ -1239,3 +1247,34 @@ def filter_queryset(self, qs):
     def paginate_queryset(self, queryset):
         # We don't need pagination for the fixed number of results.
         return queryset
+
+
+class AddonBrowserMappingView(ListAPIView):
+    authentication_classes = []
+    serializer_class = AddonBrowserMappingSerializer
+    pagination_class = LargePageNumberPagination
+
+    def get(self, request, format=None):
+        browser = self.request.query_params.get('browser', None)
+        if browser not in list(BROWSERS.values()):
+            raise exceptions.ParseError('Invalid browser parameter')
+
+        return super().get(request, format)
```

Comment:
Before returning the response from `super().get()`, add some HTTP caching. I think 24 hours should be fine ? Also add an assertion for that in `test_get` (and maybe one checking that we don't have that caching when the parameter is invalid).